File: C/1216.c
#include <stdio.h>

int main(){
    
    int n, sum = 0;
    char trash_string[99];
    double media, count = 0;
    
    while(scanf("%[^\n]", trash_string) != EOF) {
        scanf("%d\n", &n);
        sum += n;
        count++;
    }
    media = sum / count;
    printf("%.1f\n", media);

    return 0;
}

File: C#/1216.cs
using System;

class URI {
    static void Main(string[] args) {
        
        int count = 0;
        double sum = 0.0, media = 0.0;
        string trash_string;
        
        while((trash_string = Console.ReadLine()) != null){
            sum += int.Parse(Console.ReadLine());
            
            count += 1;
            
        }
        
        media = sum / count;
        
        Console.WriteLine($"{media:N1}");
    }
}